docs/pages/zero-to-one/run-it.mdx

```diff
@@ -0,0 +1,140 @@
+```bash [Terminal]
+✔ 🏗️  State Machine built
+✔ 📝 Registered STF to AppInbox
+✔ 🖖 Deployed to Vulcan
+```
+
+## Let's Run 🏃‍♂ 
+
+To be doubly sure that everything is working fine, let's try to run your application and see if it works as expected.
+
+```bash [Terminal]
+bun run src/index.ts
+```
+
+If everything is working fine, you should see the logs in the terminal, like this:
+
+```bash [Terminal]
+[Stackr] - 04/02/2024, 5:14:19 PM     LOG [InstanceLoader] ExecutorModule dependencies initialized
+[Stackr] - 04/02/2024, 5:14:19 PM     LOG [InstanceLoader] SequencerModule dependencies initialized
+[Stackr] - 04/02/2024, 5:14:19 PM     LOG [L1-Syncer] Started L1 syncer
+[Stackr] - 04/02/2024, 5:14:19 PM     LOG [Vulcan-Syncer] Started with slot time: 1000ms
+[Stackr] - 04/02/2024, 5:14:19 PM     LOG [Sequencer] Started with block time: 1000ms
+```
+
+If you see the above logs, congratulations! Your application is running successfully.
+
+## Interacting with Micro-rollup
+
+If you are using the "Counter" example, the `index.ts` file already contains the code to interact with the micro-rollup.
+```ts [index.ts]
+const inputs = {
+    timestamp: Date.now(),
+};
+
+const signature = await signMessage(wallet, UpdateCounterSchema, inputs);
+// [!code word:actionFrom]
+const incrementAction = UpdateCounterSchema.actionFrom({
+// [!code word:inputs]
+    inputs,
+    signature,
+    msgSender: wallet.address,
+});
+
+const ack = await mru.submitAction("increment", incrementAction); // [!code hl]
+```
+
+The code above is just for reference, mostly you won't be signing the message here on your backend. Rather you'll get the inputs along with signature and msgSender from the frontend and then submit the action to the micro-rollup. Using the `submitAction` method, you can submit the action to the micro-rollup.
+
+Once an action is submitted to the micro-rollup, you get an acknowledgment in the form of a `Promise` which resolves to the `Acknowledgment` object. You can use this object to track the status of the action.
+
+### Event Subscription
+
+Or you can also use the `subscribe` method to subscribe to the action status. This method takes the actionId and a callback function as arguments. The callback function is called whenever the status of the action changes.
+
+```ts [index.ts]
+mru.events.subscribe(ActionEvents.CONFIRMATION_STATUS, (data) => {
+    console.log(data);
+});
+```
+
+### Querying the State
+
+You can also query the state of your State Machines anytime using the MRU instance. As below:
+```ts [index.ts]
+const machine = mru.stateMachines.get<typeof counterMachine>("counter"); // [!code focus]
+if (!machine) {
+    throw new Error("Machine not found");
+}
+const { state } = machine; // [!code focus]
+console.log("Current state:", state);
+```
+
+
+## Conclusion
+
+In this guide, we learned how to create a micro-rollup using the Stackr SDK, deploy it to the Vulcan network, and interact with it using the SDK.
+
+## Next Steps
```

Comment:
Perhaps we can have a `Play with it` section here where we show how to use this MRU with explorer and playground

Reply:
Yes we'll add a playground in a different section

Reply:
Yes, but continuity helps here.
Dev has just built their first MRU, now would be a good time to show them the ropes with inspecting its confirmations, state and batches.